### PR TITLE
Colornames

### PR DESCRIPTION
**Pull request by ntwcklng**
I had to move the simple colornames to the bottom in order to get the tests passed.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -102,11 +102,118 @@ var mapValues = {
...
+    'darkgreen': 'dunkelgrün',
+    'darkgray': 'dunkelgrau',
+    'darkkhaki': 'dunkelkhaki',
+    'darkmagenta': 'dunkelmagenta',
+    'darkolivegreen': 'dunkelolivgrün',
+    'darkorange': 'dunkelorange',
+    'darkorchid': 'dunkleorchidee',
+    'darkred': 'dunkelrot',
+    'darksalmon': 'dunklelachsfarbe',
+    'darkseagreen': 'dunkleseegrün',
```
Comment (timche) [inline index.js]: Typo -> dunkle**s**seegrün

File: index.js [hunk 2]
```diff
@@ -102,11 +102,118 @@ var mapValues = {
...
+    'darkslateblue': 'dunkleschieferblau',
```
Comment (timche) [inline index.js]: Typo -> dunkle**s**schieferblau

**Comment by timche** [comment]
Nice PR 😄 Can you tell me where you found the translations or have you done it by yourself? I just want to make sure the translations are correct, because I'm not a expert in (german) colors 😁

**Comment by ntwcklng** [comment]
Thanks :) You can find them here: http://www.sttmedia.com/colornames
I removed the unimportant ones and changed some names so they make more sense.
With the sublime package color-highlighter you can check if all the original colornames are valid
screenshot at aug 26 12-51-59

**Comment by timche** [comment]
Alright, thanks :) I will merge it.

**Comment by timche** [comment]
What's your Twitter handle? I would like to credit you.

**Comment by ntwcklng** [comment]
Thanks, my Twitter handle: @ntwcklng

**Pull request merged**